.github/workflows/scripts/e2e/gmc_xeon_test.sh

```diff
@@ -137,7 +137,7 @@ function validate_audioqa() {
     pods_count=$(kubectl get pods -n $AUDIOQA_NAMESPACE -o jsonpath='{.items[*].metadata.name}' | wc -w)
     # expected_ready_pods, expected_external_pods, expected_total_pods
     # pods_count-1 is to exclude the client pod in this namespace
-    check_gmc_status $AUDIOQA_NAMESPACE 'audioqa' $((pods_count-1)) 0 7
```

Comment:
Hi @KfreeZ, why do you use precise number instead `pods_count` ?

Reply:
hi @zhlsunshine  because the pod count is not accurate sometimes when terminating pod is not cleared but counted

Reply:
Okay, I through that it should be okay after calling `wait_until_all_pod_ready`, if yes, could we just fix the bug for this function?

Reply:
the latest wait_until_all_pod_ready has excluded the "Terminating" pods, so there is possibility that the pods are still existing as "Terminating" pods after wait_until_all_pod_ready. Yes I can also count the pods by excluding the "Terminating" pods, but for these test case, the ready counts and total counts are fixed numbers, for chatqa for instances, after a deploy, "8/0/8" is exactly we expected.

Reply:
okay, then let's exclude the `Terminating` pods when calculating the `pods_count`.